server/plugin/webhook.go

```diff
@@ -549,12 +539,35 @@ func (p *Plugin) postIssueEvent(event *github.IssuesEvent) {
 		}
 		renderedMessage = p.sanitizeDescription(renderedMessage)
 
+		assignees := make([]string, len(issue.Assignees))
+		for index, user := range issue.Assignees {
+			assignees[index] = user.GetLogin()
+		}
+		description := ""
+		if issue.Body != nil {
+			description = *issue.Body
+		}
+
 		post := &model.Post{
 			UserId:  p.BotUserID,
-			Type:    "custom_git_issue",
 			Message: renderedMessage,
 		}
 
+		if action == constants.ActionOpened {
+			post.Type = "custom_git_issue"
+			post.Props = map[string]interface{}{
+				constants.TitleForProps:       *issue.Title,
+				constants.IssueURLForProps:    *issue.HTMLURL,
+				constants.IssueNumberForProps: *issue.Number,
+				constants.DescriptionForProps: description,
```

Comment:
So the current conditions on this is to show these controls on a post created by the plugin, specifically for when an issue is opened.

I think it would be useful to have for PRs and when labeling issues. And possibly in the bot DMs as well, though that could get noisy. @hanzei What are your thoughts on this?

Reply:
1/5 that it's handy to have actions attached to the DM notification as this allows user to answer them directly in MM. Maybe we even add the buttons to all posts?

@asatkinson What do you think?